### ISSUE TITLE
Copy change: Chapter 6 outro

### ISSUE THREAD

**Issue by satsie** [saving-satoshi/saving-satoshi#947]
At the end of chapter 6, the outro mentions getting off Vanderpoole's private island. I don't think we actually go to the island in this chapter, we get the money so Mike Ramen can go.

![Screenshot from 2024-05-15 23-25-56](https://github.com/saving-satoshi/saving-satoshi/assets/1823216/83ad25e3-8b9e-4ab7-979f-ffdd40ea33a5)

I'm happy to propose a copy change for this but I haven't made it through the rest of the game (so far), so I can revisit this once I have a better idea of how the rest of the story goes.

**Comment by benalleng** [comment]
yeah this is BEFORE we get to the island. Maybe, `You successfully built a transaction from scratch to pay Mike Ramen for his help. Now that we are done let's see what we can find on Vanderpoole's island...`

**Comment by benalleng** [comment]
closed by #957
